### PR TITLE
docs: add link to Bridge In Tech zulip stream in CONTRIBUTING.md

### LINKED ISSUE #204: Docs: Add link to zulip stream in CONTRIBUTING.md
### Describe the bug

The link to project's zulip stream is missing in CONTRIBUTING.md. 

### To Reproduce
1. Go to [contributing.md](https://github.com/anitab-org/bridge-in-tech-web/blob/develop/.github/CONTRIBUTING.md)
2. See error

### Expected behavior

Add link to zulip stream in CONTRIBUTING.md

### Screenshots

bit-cont

### Desktop (please complete the following information):
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by battuAshita**
### Description

Fixes #204

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by devkapilbansal — DISMISSED** [review]
LGTM :+1: 
Thanks @battuAshita for this

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 
-* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
+* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). You can find the Bridge in Tech stream [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech).
```
Comment (codesankalp) [inline .github/CONTRIBUTING.md]:
```suggestion
* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech).
```

**Review by codesankalp — CHANGES_REQUESTED** [review]
@battuAshita Thanks for your contribution :rocket: 
It will be ready to merge if you will make the suggested changes.
The changes are small but important as it resembles more with other repos `CONTRIBUTING.md`

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (battuAshita) [reply .github/CONTRIBUTING.md]: Will make these changes :)

**Review by codesankalp — APPROVED** [review]
LGTM :tada: , Thanks again @battuAshita for your contribution.

**Review by Amulya-coder — APPROVED** [review]
LGTM @battuAshita Thanks for your contribution 🎉

**Review by isabelcosta — APPROVED** [review]
Great work @battuAshita 🎉

**Pull request merged**